src/staticanalysis/bot/static_analysis_bot/infer/__init__.py

```diff
@@ -63,7 +63,8 @@ def setup(index, job_name='linux64-infer', revision='latest',
             try:
                 artifact_url = index.buildSignedUrl('findArtifactFromTask',
                                                     indexPath=namespace,
-                                                    name=artifact)
+                                                    name=artifact,
+                                                    expiration=7200)
```

Comment:
120 minutes seems a long time to download these three artifacts, would a bit shorter expiration work for you anyways?

Reply:
I wouldn't go lower than one hour, would this be something acceptable though?